features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -258,6 +258,11 @@
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.freebox/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-freemobilesms" description="FreeMobile SMS API Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.freemobilesms/${project.version}</bundle>
```

Comment:
Use "mvn:org.openhab.addons.bundles/" as seen on other entries in this file

Reply:
Done